### DISCUSSION TITLE
when will PS 7.4.3 be available through Microsoft Update Catalogue

### DISCUSSION THREAD

**Discussion by shertaeg** [PowerShell/PowerShell#24022] [General]
Hello all,

I was just wondering, _as I already went through Microsoft Support-pain, which couldn't help_, is there a plan on making PS 7.4.3 available through Microsoft Update Catalogue and therefore WSUS? It was released on June 18th and still isn't available through said channels.

> https://www.catalog.update.microsoft.com/Search.aspx?q=powershell

I'd love to update my systems with least effort.
Thank you

**Comment by StevenBucher98** [comment]
Unfortunately we have hit some issues with the latest release and is delayed. Unfortunately do not have an ETA yet

**Comment by shertaeg** [comment]
Hi Steven, thank you for your answer.
What happened to SFI? scnr.

As v7.4.4 is already released through github, please update the catalog as well as soon as possible. Vulnerability scanners will already mark 7.4.3 as vulnerable/old, even as it was just released on monday in the Microsoft Update catalog.

**Reply by StevenBucher98** [reply]
I am not sure what you are referring to with SFI and scnr, regardless there is typically [a 2 week delay on our MU release](https://learn.microsoft.com/en-us/powershell/scripting/install/microsoft-update-faq?view=powershell-7.4#how-soon-after-release-are-updates-advertised-by-microsoft-update) from when we release on GitHub, this is part of the process and not something we can speed up. We are having a focus in security lately and so are improving some of our release tests that may also create a delay in our MU releases.

**Comment by kilasuit** [comment]
Marking resolved based on this install dates for 7.4.3 & 7.4.4

![image](https://github.com/user-attachments/assets/c6e6edc1-4299-49dc-bc1a-5f0e8a9b4956)
